resources/views/auth/register.blade.php

```diff
@@ -36,73 +36,130 @@ class="flow bg-indigo-500 hover:bg-indigo-700 text-white font-bold py-2 px-4 rou
                              autocomplete="username"/>
                 </div>
 
+                <div id="institutionDiv">
+                    @if(!old('company_name'))
+                        <div class="mt-4">
```

Comment:
Maybe put some indication here that people don't have to fill this in themselves. Or that is automatically gotten from the email address. Could be a greyed out input field. That might also mean that the input is not necessarily required to registered.